src/Umbraco.Infrastructure/PropertyEditors/ValueConverters/BlockListPropertyValueConverter.cs

```diff
@@ -3,24 +3,28 @@
 
 using System.Reflection;
 using Umbraco.Cms.Core.Logging;
+using Umbraco.Cms.Core.Models;
 using Umbraco.Cms.Core.Models.Blocks;
 using Umbraco.Cms.Core.Models.PublishedContent;
+using Umbraco.Cms.Core.Services;
 using Umbraco.Extensions;
 
 namespace Umbraco.Cms.Core.PropertyEditors.ValueConverters;
 
 [DefaultPropertyValueConverter(typeof(JsonValueConverter))]
 public class BlockListPropertyValueConverter : PropertyValueConverterBase
 {
+    private readonly IContentTypeService _contentTypeService;
     private readonly BlockEditorConverter _blockConverter;
     private readonly BlockListEditorDataConverter _blockListEditorDataConverter;
     private readonly IProfilingLogger _proflog;
 
-    public BlockListPropertyValueConverter(IProfilingLogger proflog, BlockEditorConverter blockConverter)
+    public BlockListPropertyValueConverter(IProfilingLogger proflog, BlockEditorConverter blockConverter, IContentTypeService contentTypeService)
     {
         _proflog = proflog;
         _blockConverter = blockConverter;
         _blockListEditorDataConverter = new BlockListEditorDataConverter();
+        _contentTypeService = contentTypeService;
     }
```

Comment:
This is a breaking change, so we can't do it, I suggest making a new constructor instead, and then calling `: this()` from the old one instead 👍

Reply:
Like https://github.com/umbraco/Umbraco-CMS/pull/13216/commits/a08bc58abd8601a68aa4d432f6fa1d6197617404 right?

Reply:
Yep 🙌